packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -890,6 +867,34 @@ export class ConversationAPI {
    * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/updateConversationAccess
    */
   public async putAccess(
+    conversationId: QualifiedId | string,
```

Comment:
Let's force the consumer to use qualified ids from now on. 
```suggestion
    conversationId: QualifiedId,
```

Reply:
Actually, since only the url changes but the rest is exactly the same, let's just have a small helper that generates qualified/non-qualified base urls. 

```ts
  private generateBaseConversationUrl(conversationId: QualifiedId): string {
    return this.backendFeatures.federationEndpoints
      ? `${ConversationAPI.URL.CONVERSATIONS}/${conversationId.domain}/${conversationId.id}`
      : `${ConversationAPI.URL.CONVERSATIONS}/${conversationId.id}`;
  }
```

and we can use this helper everywhere we need to support both unqualified and qualified urls. WDYT?